src/executables/master_server_monitor.cpp

```diff
@@ -184,22 +179,27 @@ server_monitor::revive_server (const std::string &server_name)
 	  out_pid = try_revive_server (entry->second.get_exec_path(), entry->second.get_argv());
 	  if (out_pid == -1)
 	    {
-	      fprintf (stdout, "[SERVER_REVIVE_DEBUG] : Fork at server revive failed. exec_path : %s, args : %s\n",
-		       entry->second.get_exec_path().c_str(), entry->second.get_argv()[0]);
+	      _er_log_debug (ARG_FILE_LINE,
+			     "[Server Monitor] %s : Failed to execute server. Server monitor will try to execute server again. (pid : %d)\n",
+			     entry->first.c_str(), entry->second.get_pid());
 	      produce_job_internal (job_type::REVIVE_SERVER, -1, "", "", entry->first);
 	    }
 	  else
 	    {
 	      entry->second.set_pid (out_pid);
+	      _er_log_debug (ARG_FILE_LINE,
+			     "[Server Monitor] %s : Server monitor is waiting for server to be registered. (pid : %d)\n",
+			     entry->first.c_str(), entry->second.get_pid());
 	      produce_job_internal (job_type::CONFIRM_REVIVE_SERVER, -1, "", "",
 				    entry->first);
 	    }
 	  return;
 	}
       else
 	{
-	  fprintf (stdout, "[SERVER_REVIVE_DEBUG] : Process failure repeated within a short period of time. pid : %d\n",
-		   entry->second.get_pid());
+	  _er_log_debug (ARG_FILE_LINE,
+			 "[Server Monitor] %s : Process failure repeated within a short period of time. Server process will be terminated. (pid : %d)\n",
+			 entry->first.c_str(), entry->second.get_pid());
```

Comment:
a short period of time 메시지 timediff 값을 사람이 볼 수 있는 형태로 추가
- ex) a short period of time(60s)

```suggestion
	  _er_log_debug (ARG_FILE_LINE,
			 "[Server Monitor] [%s] Process failure repeated within a short period of time. Server process will be terminated. (pid : %d)\n",
			 entry->first.c_str(), entry->second.get_pid());
```

Reply:
[Server Monitor] [%s] Process failure repeated within a short period of time (%d sec). Server process will be terminated. (pid : %d) 로 수정하였습니다.